recipe/meta.yaml

```diff
@@ -36,7 +37,7 @@ test:
   requires:
     - pip
     - gdal
```

Comment:
Why is `gdal` required to run the tests?

Reply:
I'm just going to merge for now, but when you update to the current release on pypi (25.1), then maybe test out whether it's actually needed

Reply:
@synapticarbors thank you. I will do. Much appreciated.